### PR TITLE
Poland mkuran.pl updates [SOURCES]

### PR DESCRIPTION
**Pull request by A-dead-pixel**
As mostly stated in the commit messages:

feat: add more feeds from mkuran.pl [SOURCES] (the only available one which I haven't added is the Kielce one, as they have a funky license (see mkuran.pl/gtfs).)
- pl-kujawsko-pomorskie-miejski-zaklad-komunikacji-w-toruniu-mzk-torun-gtfs-2083.json
- pl-pomorskie-miejski-zaklad-komunikacji-w-wejherowie-mzk-wejherowo-gtfs-2084
- pl-zachodniopomorskie-komunikacja-autobusowa-swinoujscie-ka-swinoujscie-gtfs-2085.json
- pl-warminsko-mazurskie-miejski-zaklad-komunikacji-w-elku-mzk-elk-gtfs-2086
- pl-unknown-polregio-gtfs-2087
- pl-unknown-polskie-koleje-panstwowe-intercity-pkp-intercity-gtfs-2088

fix: update URLs for feeds from mkuran.pl [SOURCES] (`/feed/*` -> `/gtfs/*`)
- pl-mazowieckie-warszawska-kolej-dojazdowa-wkd-gtfs-863.json
- pl-mazowieckie-warszawski-transport-publiczny-ztm-warszawa-gtfs-1008.json
- pl-mazowieckie-mzdik-radom-gtfs-1015.json

fix: update feeds removed from mkuran.pl [SOURCES]
- pl-pomorskie-zarzad-transportu-miejskiego-gdansk-ztm-gdansk-gtfs-1014.json (this one had data on Gdynia and Gdańsk combined, but only Gdańsk publishes an up-to-date GTFS feed, so now this feed doesn't include Gdynia)
- pl-mazowieckie-komunikacja-miejska-lomianki-gtfs-1123.json (there is no replacement, so I marked it as `inactive`)

### CODE REVIEW

**Comment by emmambd** [comment]
Thanks for this amazing review of available Poland data @A-dead-pixel! 

We don't change the URL for feeds, so that there's a consistent definition of what a feed is over time. Because of this, there's two options: 

#1: You could update this PR to create new feeds for the new URLs, set the status of the old feeds to "deprecated", and add a redirect ID to the new feeds that replace the old ones (everything in fix: update URLs for feeds from mkuran.pl [SOURCES] (/feed/* -> /gtfs/*) + the changes to 1014) 

#2: You could remove the feed updates from this PR, and I could do them instead to make the process simpler for you. This PR would solely consist of the new feeds, and I can add you as a reviewer to the new PR for feed updates.

**Comment by A-dead-pixel** [comment]
Ok, I will update the PR shortly and do the feed updates myself. It does indeed make more sense, especially for 1014.

**Comment by A-dead-pixel** [comment]
I deprecated the feeds and set the redirects to new ones. The only change in the first commit is the message - it had stray `.json`s.

**Comment by A-dead-pixel** [comment]
I fixed the conflict between feeds with ids 2083 (one in `main` and the other here) and rebased onto `main`.

File: catalogs/sources/gtfs/schedule/pl-pomorskie-zarzad-transportu-miejskiego-gdansk-ztm-gdansk-gtfs-1014.json [hunk 1]
```diff
@@ -18,5 +19,11 @@
         "direct_download": "https://mkuran.pl/gtfs/tristar.zip",
         "latest": "https://storage.googleapis.com/storage/v1/b/mdb-latest/o/pl-pomorskie-zarzad-transportu-miejskiego-gdansk-ztm-gdansk-gtfs-1014.zip?alt=media",
         "license": "http://www.opendefinition.org/licenses/cc-by"
-    }
+    },
```
Comment (emmambd) [inline catalogs/sources/gtfs/schedule/pl-pomorskie-zarzad-transportu-miejskiego-gdansk-ztm-gdansk-gtfs-1014.json:22]: @A-dead-pixel Upon further investigation, it looks like there is a replacement feed for Gdynia: http://api.zdiz.gdynia.pl/pt/gtfs.zip

Could you add it and then also include it in the redirect list?

Comment (A-dead-pixel) [reply catalogs/sources/gtfs/schedule/pl-pomorskie-zarzad-transportu-miejskiego-gdansk-ztm-gdansk-gtfs-1014.json:22]: Ah, you're right! This [site](https://otwartedane.gdynia.pl/pl/dataset/informacje-o-rozkladach-jazdy-i-lokalizacji-przystankow) with more info about that feed misled me, as it says the last update was in 2023, though the file contains recent data.
I added it.

**Review by emmambd — APPROVED** [review]
Thanks for this very thorough overview of available Poland data, @A-dead-pixel! Your contribution is so valuable.

**Pull request merged**